src/main/java/com/ffxivcensus/gatherer/player/PlayerBean.java

```diff
@@ -3,93 +3,155 @@
 import java.util.Date;
 import java.util.List;
 
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import com.ffxivcensus.gatherer.util.StringListConverter;
+
 /**
  * Object class to represent a Character/Player. This class specifies the attributes and behaviour of a player object.
  *
  * @author Matthew Hillier
  * @since v1.0
  * @see PlayerBuilder
  */
+@Entity
+@Table(name = "tblplayers")
 public class PlayerBean {
+    private static final String NOT_AVAILABLE = "N/A";
+    @Id
     private int id;
-    private String realm;
-    private String playerName;
-    private String race;
-    private String gender;
-    private String grandCompany;
-    private String freeCompany;
-    private int lvlGladiator;
-    private int lvlPugilist;
-    private int lvlMarauder;
-    private int lvlLancer;
-    private int lvlArcher;
-    private int lvlRogue;
-    private int lvlConjurer;
-    private int lvlThaumaturge;
-    private int lvlArcanist;
-    private int lvlDarkKnight;
-    private int lvlMachinist;
-    private int lvlAstrologian;
-    private int lvlScholar;
-    private int lvlRedMage;
-    private int lvlSamurai;
-    private int lvlCarpenter;
-    private int lvlBlacksmith;
-    private int lvlArmorer;
-    private int lvlGoldsmith;
-    private int lvlLeatherworker;
-    private int lvlWeaver;
-    private int lvlAlchemist;
-    private int lvlCulinarian;
-    private int lvlMiner;
-    private int lvlBotanist;
-    private int lvlFisher;
+    private String realm = NOT_AVAILABLE;
+    private String name = NOT_AVAILABLE;
+    private String race = NOT_AVAILABLE;
+    private String gender = NOT_AVAILABLE;
+    private String grandCompany = NOT_AVAILABLE;
+    private String freeCompany = NOT_AVAILABLE;
+    private int levelGladiator;
+    private int levelPugilist;
+    private int levelMarauder;
+    private int levelLancer;
+    private int levelArcher;
+    private int levelRogue;
+    private int levelConjurer;
+    private int levelThaumaturge;
+    private int levelArcanist;
+    private int levelDarkknight;
```

Comment:
The default behaviour of Spring JPA is to split camel-case works in java into lower-case words with underscores in the database. As a result, I've changed the bean names here to conform to these assumptions so that we don't have to do lots of mapping.

Note, this does mean "Dark Knight" and "Red Mage" are captured as the slightly-awkward "Darkknight" and "Redmage" due to the existing table layout.